meta-mbl-distro/scripts/create-update-payload

```diff
@@ -72,153 +74,377 @@ bitbakepath = scriptpath.add_bitbake_lib_path()
 import bb.tinfoil
 import argparse_oe
 
+import pdb
+
+MBL_BOOT_ID = "BOOT"
+PAYLOAD_FORMAT_VERSION = b"1"
+
 
 def parse_args():
     """Parse the command line arguments."""
     parser = argparse_oe.ArgumentParser(description=__doc__)
-    group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument(
-        "-f",
-        "--filenames",
-        nargs="+",
-        help="Space separated list of filenames of update components.",
-    )
-    group.add_argument(
+    parser.add_argument(
         "-b",
-        "--bootloader-slots",
-        nargs="+",
-        help="Space separated list of bootloader slot numbers. e.g 1 would"
-        " correspond to bootloader slot one.",
+        "--bootloader-component",
+        dest="bootloader_components",
+        help="Add bootloader component N to the payload.",
+        metavar="N",
+        action="append",
```

Comment:
I don't get why this was changed. Previously, because I was using `nargs="+"`, I could just give a space separated list of the bootloader slots, e.g `-b 1 2`, with this change we have to keep repeating the `-b` option e.g `-b 1 -b 2`. I'm changing this back to use `nargs="+"`